src/components/uielements/assets/assetMenu/AssetMenu.tsx

```diff
@@ -25,34 +25,25 @@ const AssetMenu: React.FC<Props> = (props: Props): JSX.Element => {
   const { assetData, asset, priceIndex = {}, withSearch, searchDisable = [], onSelect = () => {} } = props
 
   const filteredData = useMemo(
-    () =>
-      assetData.filter((item) => {
-        const { ticker = '' } = item.asset
-        return ticker !== asset?.ticker
-      }),
+    () => assetData.filter((item) => item.asset.ticker !== asset?.ticker),
```

Comment:
is still asset a nullable value

Reply:
You are right, it's a `NonNullable` asset now

Reply:
so maybe we dont need `asset?` ?

Reply:
I think it would be more safety to pass an `Asset` we already have instead of a `string` into this component. A `string` could be anything else, but no guarantee to have a `ticker` of an `Asset`.

Reply:
Or did you mean the `?.` operator for optional `ticker`. Yes, it can be removed then.